go.mod

```diff
@@ -27,6 +25,8 @@ require (
 	gopkg.in/dixonwille/wmenu.v4 v4.0.2
 )
 
+require github.com/sabhiram/go-gitignore v0.0.0-20210923224102-525f6e181f06
```

Comment:
I don't have much experience with go so I'm not sure whether what it added here is okay or should be done any differently.

Reply:
I don't think it's a problem, but I'll just merge it in with the rest of them to look a bit nicer